dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/EntryVersionHelper.java

```diff
@@ -134,15 +136,20 @@ static void stripContent(List<? extends Entry> entries, AbstractDockstoreDAO dao
             dao.evict(entry);
             // clear users which are also lazy loaded
             entry.setUsers(null);
-            // need to have this evicted so that hibernate does not actually delete the tags and users
```

Comment:
Have a question. if I understand correctly, the deleted code was relying on eager loading but prevents deleting content when stripping it out before serialization. 

`removeSourceFilesFromEntry` seems to save the current state and then do the eviction. But why is a eviction no longer needed here specifically?

Reply:
With my changes, the sourcefiles are not always lazily loaded. When you refresh an entry or edit a hosted one, the sourcefiles end up getting loaded, so that's why I created the new function to remove the sourcefiles. But the sourcefiles aren't loaded whenever `stripContent()` was called.